docs/diagnostics/experimental-apis/OTEL1003.md

```diff
@@ -11,14 +11,10 @@ Experimental APIs may be changed or removed in the future.
 
 ## Details
 
-The OpenTelemetry Specification defines the
-[cardinality limit](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#cardinality-limits)
-of a metric can be set by the matching view.
-
 From the specification:
 
-> The cardinality limit for an aggregation is defined in one of three ways:
-> A view with criteria matching the instrument an aggregation is created for has
+> The cardinality limit for an aggregation is defined in one of three ways: A
+> view with criteria matching the instrument an aggregation is created for has
 > an aggregation_cardinality_limit value defined for the stream, that value
 > SHOULD be used. If there is no matching view, but the MetricReader defines a
 > default cardinality limit value based on the instrument an aggregation is
```

Comment:
> If there is no matching view, but the MetricReader defines a
> default cardinality limit value based on the instrument an aggregation is
> created for, that value SHOULD be used. "

Shall we remove this since it's being obsoleted?

Reply:
I left it alone because it is coming from the spec. But I added a note that we don't support the `MetricReader` bit. Thoughts?

Reply:
sounds good to me!